provider_shopper/lib/screens/cart.dart

```diff
@@ -46,6 +46,12 @@ class _CartList extends StatelessWidget {
       itemCount: cart.items.length,
       itemBuilder: (context, index) => ListTile(
         leading: Icon(Icons.done),
+        trailing: IconButton(
+          icon: Icon(Icons.delete),
```

Comment:
It would be more correct to use a "remove" icon for the button, rather than a "delete" icon.

Consider this one, for example: https://material.io/resources/icons/?icon=remove_circle_outline&style=baseline